src/protocols/thresholdsig/zilliqa_schnorr.rs

```diff
@@ -43,6 +43,11 @@ pub struct KeyGenBroadcastMessage1 {
     com: BigInt,
 }
 
+pub struct KeyGenMessage2 {
```

Comment:
Can we change it to `KeyGenBroadcastMessage2` (not the `Broadcast`) for consistency?